### ISSUE TITLE
Weird line break issues with SVG

### ISSUE THREAD

**Issue by graipher** [mixn/carbon-now-cli#175]
**Describe the bug**
When saving as SVG, some lines are weirdly broken. `widthAdjustment` is `true`. The PNG version is fine. Only happens with `chromium` driver, not `firefox`.

**To Reproduce**

1. Run `carbon-now` with…
`config.json`
```js
{
  "latest-preset": {
    "theme": "material",
    "backgroundColor": "#00000000",
    "windowTheme": "none",
    "windowControls": false,
    "fontFamily": "Fira Code",
    "fontSize": "18px",
    "lineNumbers": false,
    "firstLineNumber": 1,
    "dropShadow": false,
    "dropShadowOffsetY": "20px",
    "dropShadowBlurRadius": "68px",
    "selectedLines": "*",
    "widthAdjustment": true,
    "lineHeight": "133%",
    "paddingVertical": "2px",
    "paddingHorizontal": "2px",
    "squaredImage": false,
    "watermark": false,
    "exportSize": "1x",
    "type": "svg",
    "language": "python"
  }
}
```
and
`generics.py`
```python
from typing import Generic, TypeVar

T = TypeVar("T")

def f(a: T) -> T:
	return a

class A(Generic[T]):
	a: T
```

`carbon-now generics.py --engine chromium`

3. Wait for image to download…

**Expected behavior**
SVG and PNG output should be the same. There should be no weird linebreaks that are not present in the file.

**Screenshots**
SVG:
![Generics_1](https://github.com/mixn/carbon-now-cli/assets/984262/79293f6e-bd5a-4844-bbc4-289c075f0a4e)
PNG:
![Generics_1](https://github.com/mixn/carbon-now-cli/assets/984262/daf92fb8-d957-4c78-a191-3035baf86e7a)

**Info (please complete the following information):**

- OS [e.g. macOS, Linux, Windows, iOS]: Linux
- `--engine` used [e.g. `chromium`, `firefox`, `webkit`]: `chromium`
- Carbon URL (run with `--open-in-browser` to see) [e.g. carbon.now.sh?bg=pink]: browser does not open

**Comment by mixn** [comment]
@graipher Thank you for opening; sorry I missed this earlier last month. I’ll investigate further this weekend, but I think this is an error on Carbon’s part (not the CLI), since I remember having different outputs when implementing `--engine` as part of `v2` and I even had to use `--engine firefox` for some of the screenshots of the docs. 

I will report back, though; thanks for your patience! 🙂

**Comment by mixn** [comment]
Been quite swamped since the beginning of the year, but I will tackle a new release of the CLI soon and, amongst other things, address this (somehow.) 🙂 Thanks for your patience, appreciated! @graipher

**Comment by mixn** [comment]
Apologies that it took me so long to get to this. 😅 Once I started it was all straightforward and fast, I should have just started earlier. 😄 That being said, I just shipped [`v2.1`](https://github.com/mixn/carbon-now-cli/releases/tag/v2.1.0). 🎉

This (technically) always worked, see more details in [this extensive comment I wrote in #207](https://github.com/mixn/carbon-now-cli/issues/207#issuecomment-2538467340).

### For completeness-purposes:

The commands I ran for PNGs:
```
carbon-now generics.py --engine chromium --save-as engine-chromium --settings '{"theme":"material"}'
carbon-now generics.py --engine firefox --save-as engine-firefox --settings '{"theme":"material"}'
carbon-now generics.py --engine webkit --save-as engine-webkit --settings '{"theme":"material"}'
```

The commands I ran for SVGs:
```
carbon-now generics.py --engine chromium --save-as engine-chromium-svg --settings '{"theme":"material","type":"svg","widthAdjustment":"false"}'
carbon-now generics.py --engine firefox --save-as engine-firefox-svg --settings '{"theme":"material","type":"svg","widthAdjustment":"false"}'
carbon-now generics.py --engine webkit --save-as engine-webkit-svg --settings '{"theme":"material","type":"svg","widthAdjustment":"false"}'
```

### Results

#### PNGs
![engine-chromium](https://github.com/user-attachments/assets/7dc868a2-fc3e-4784-9a49-b1e512de9407)
![engine-firefox](https://github.com/user-attachments/assets/5b2be851-e3d5-44d3-bff1-b7b199a1c280)
![engine-webkit](https://github.com/user-attachments/assets/170a93bf-d919-4a35-9c73-e8be4f523e22)

#### SVGs
